src/Sulu/Bundle/PreviewBundle/DependencyInjection/SuluPreviewExtension.php

```diff
@@ -36,10 +36,10 @@ public function load(array $configs, ContainerBuilder $container)
 
     public function prepend(ContainerBuilder $container)
     {
-        if ($container->hasExtension('doctrine_cache')) {
-            $configs = $container->getExtensionConfig($this->getAlias());
-            $config = $this->processConfiguration(new Configuration(), $configs);
+        $configs = $container->getExtensionConfig($this->getAlias());
+        $config = $this->processConfiguration(new Configuration(), $configs);
 
+        if ($config['cache']['type'] && $container->hasExtension('doctrine_cache')) {
```

Comment:
Should we maybe throw an error when the `doctrine_cache` extension does not exist but the `cache.type` configuration is set? I just tried to reuse the old configuration, and tripped because of that.